### PR TITLE
Intuitive Jobnames, consistency across tests and slack notifications for all tests

### PR DESCRIPTION
**Pull request by svetsa-rh**
Updated all current tests so it's easy to know what tests are running looking at test name, have consistent names across the board and update all tests missing notifications to notify results to slack.

Next: Clean up duplicate jobs or no longer applicable jobs in a separate upcoming PR.

### CODE REVIEW

**Comment by svetsa-rh** [comment]
@paigerube14 @jtaleric This PR is ready for review. Can you ptal?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86-sno.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-aws-control-plane-sno-periodic
+- as: perfscale-aws-control-plane-sno-3nodes-periodic
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86-sno.yaml:24]: sno means it'll only have 1 master/worker node by default

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-loaded-upgrade-from-4.12.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-loaded-upgrade-aws
+- as: perfscale-aws-5nodes-loaded-upgrade-412to413to414-periodic
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-loaded-upgrade-from-4.12.yaml:39]: here I think it would be ok to just put 412 to 414 because you have to go through 4.13 anyways

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-sno.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-aws-control-plane-sno-periodic
+- as: perfscale-aws-control-plane-sno-3nodes-periodic
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-sno.yaml:24]: same as other sno comment

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml [hunk 4]
```diff
@@ -37,7 +37,7 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     workflow: openshift-qe-control-plane
   timeout: 5h0m0s
-- as: ocp-qe-perfscale-aws-cp-24nodes-periodic
+- as: perfscale-aws-control-plane-24nodes-periodic
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml:40]: I would be ok with keeping this as cp to shorten the name

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml [hunk 5]
```diff
@@ -87,8 +87,38 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     - chain: workers-scale
     workflow: openshift-qe-node-density-heavy
+- as: perfscale-rosa-multiaz-data-path-9nodes-periodic
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:90]: should we put this in a separate pr since we are focusing on renaming

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml [hunk 6]
```diff
@@ -21,15 +21,19 @@ resources:
...
     env:
       CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
       OPENSHIFT_VERSION: "4.12"
+      REPLICAS: "24"
     test:
-    - chain: openshift-qe-data-path-tests
+    - ref: openshift-qe-cluster-density-v2
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml:36]: think we would want to keep as data path tests here?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml [hunk 7]
```diff
@@ -21,15 +21,19 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: perfscale-rosa-network-perf-periodic
-  cron: 0 8 * * 5
+- as: perfscale-rosa-multiaz-cluster-density-v2-24nodes-periodic
+  cron: 0 8 * * 4
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml:25]: all other tests in this file you are setting to not run, do you want to set this one to not run as well?

**Review by jtaleric — APPROVED** [review]

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-loaded-upgrade-from-4.12.yaml (hunk 2 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-loaded-upgrade-from-4.12.yaml:39]: true, will leave this anyway unless you feel strongly about it

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml (hunk 4 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml:40]: I expanded it to control-plane as some tests already had `control-plane` mentioned and also for any folks outside of the perfscale group it might be more intuitive

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml (hunk 6 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml:36]: Roshni suggested that we run only control-plane tests.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml (hunk 7 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml:25]: we want to run cluster-density-v2 test on a cron basis and not run others on a cron basis

**Comment by svetsa-rh** [comment]
@paigerube14 Can you ptal again? I updated based on feedback.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml (hunk 5 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml]: Looks like that got in there accidentally, I removed it.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-sno.yaml [hunk 8]
```diff
@@ -21,7 +21,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-aws-control-plane-sno-periodic
+- as: perfscale-aws-control-plane-sno-periodic
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-sno.yaml:24]: this test and the 4.13 sno test are named the exact same. did you add version to all test names?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml [hunk 9]
```diff
@@ -87,8 +87,22 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     - chain: workers-scale
     workflow: openshift-qe-node-density-heavy
+- as: perfscale-gcp-network-policy-5nodes-periodic
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:90]: can we move this test to a new file?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml (hunk 7 above)
Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml:25]: should we run all of the control plane tests not just cluster-density?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml [hunk 10]
```diff
@@ -55,28 +59,24 @@ tests:
...
     workflow: rosa-aws-sts
-- as: perfscale-rosa-multiaz-node-density-v2-24nodes-periodic
-  cron: 0 8 * * 3
+- always_run: false
+  as: perfscale-rosa-data-path-3nodes
   steps:
     cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: nightly
-      COMPUTE_MACHINE_TYPE: m5.xlarge
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml:68]: should we just keep these variable for future runs (if someone was to turn this back on) would be consistent configurations

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml (hunk 9 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:90]: new file? why? It's 4.13 test.

Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:90]: its a whole new section, thought we should keep this to only renaming

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-sno.yaml (hunk 8 above)
Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-sno.yaml:24]: will the periodic names add the version?

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-sno.yaml:24]: not sure I understand. we don't mention version number in the test names. may be I am not understanding the ask here

Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-sno.yaml:24]: don't know if its an issue that this test name and the sno test for 4.13 are the exact same name

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml (hunk 10 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml:68]: I actually didn't make any changes to the test.
And not sure what all variables are missing for it, will just delete the test for now as I think it was added very early stages and is a non periodic test

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml (hunk 9 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:90]: Since I was removing cron jobs from 4.12, I moved it to 4.13.

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:90]: this PR is more than renaming, including cleaning up of some jobs

**Comment by svetsa-rh** [comment]
@paigerube14 I think it needs lgtm label
